js/pathfinding.js

```diff
@@ -69,7 +115,14 @@ function getCachedNodes(token) {
 		cacheData.tokenWidth = token.data.width;
 		cacheData.tokenHeight = token.data.height;
```

Comment:
There's a `getTokenSize` function in Drag Ruler that you could utilize, which does basically the same thing (on hex and square grids)